packages/desktop-gui/cypress/integration/setup_project_modal_spec.js

```diff
@@ -447,13 +447,20 @@ describe('Set Up Project', function () {
       cy.get('.modal').contains('Log In to Dashboard')
     })
 
+    it('closes login modal', () => {
+      cy.get('.modal').contains('Log In to Dashboard')
+      cy.get('.close').click()
+      cy.get('.btn').contains('Set up project').click()
+    })
+
     describe('when login succeeds', function () {
       beforeEach(function () {
         cy.stub(this.ipc, 'beginAuth').resolves(this.user)
         cy.contains('button', 'Log In to Dashboard').click()
       })
 
       it('shows setup', () => {
+        cy.get('.login-content > .btn').click()
```

Comment:
why this addition?

Reply:
previously, this test would always pass because both dialogs are actually on screen at the same time, but the login dialog obscures the setup dialog.

with this PR, the setup dialog opens *after* the login dialog closes, so the user actually needs to click "continue" for the `<h4>` to exist in the dom.